<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenerateTilesToolClass</class>
 <widget class="QDialog" name="GenerateTilesToolClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate tiles</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>541</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>131</width>
        <height>31</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="runButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="closeButton">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Width:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Height:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>330</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tiles Y:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tiles X:</string>
   </property>
  </widget>
  <widget class="QLabel" name="mapTileHeightLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tile height:</string>
   </property>
  </widget>
  <widget class="QLabel" name="mapTileWidthLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tile width:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mapWidthLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="mapHeightLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="tilesXLineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="tilesYLineEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>330</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="mapTileWidthLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="mapTileHeightLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>110</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="targetPathLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Target path:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="targetPathPushButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>300</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mapPaddingLineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Padding:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Offset X:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tileOffsetWidthLineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>360</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="tileOffsetHeightLineEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>360</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>360</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Offset Y:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Suffix:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tileSuffixLineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>390</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="DEM_DatasetPushButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>160</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>DEM dataset:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DEM_DatasetLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="mapDatasetPushButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>20</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Map dataset:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mapDatasetLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>401</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="DEM_TileWidthLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tile width:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Padding:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DEM_PaddingLineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Width:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DEM_TileWidthLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="DEM_TileHeightLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="DEM_WidthLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="DEM_HeightLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="DEM_TileHeightLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tile height:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Height:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>targetPathPushButton</sender>
   <signal>clicked()</signal>
   <receiver>GenerateTilesToolClass</receiver>
   <slot>setTargetPath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tilesXLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>GenerateTilesToolClass</receiver>
   <slot>refresh(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tilesYLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>GenerateTilesToolClass</receiver>
   <slot>refresh(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mapPaddingLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>GenerateTilesToolClass</receiver>
   <slot>refresh(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DEM_DatasetPushButton</sender>
   <signal>clicked()</signal>
   <receiver>GenerateTilesToolClass</receiver>
   <slot>openDEM_Dataset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>535</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mapDatasetPushButton</sender>
   <signal>clicked()</signal>
   <receiver>GenerateTilesToolClass</receiver>
   <slot>openMapDataset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>535</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>GenerateTilesToolClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>512</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>runButton</sender>
   <signal>clicked()</signal>
   <receiver>GenerateTilesToolClass</receiver>
   <slot>start()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DEM_PaddingLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>GenerateTilesToolClass</receiver>
   <slot>refresh(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refresh(QString)</slot>
  <slot>setTargetPath()</slot>
  <slot>openMapDataset()</slot>
  <slot>openDEM_Dataset()</slot>
  <slot>start()</slot>
 </slots>
</ui>
